package.json

```diff
@@ -84,9 +87,13 @@
     "lint:fix": "yarn run lint --fix"
   },
   "lint-staged": {
-    "{packages,examples}/**/*.{js,jsx,ts,tsx,json,css,scss,md}": [
+    "{packages,examples}/**/*.{json,css,scss,md}": [
```

Comment:
I ended up keeping these rules more explicitly split. It turns out that eslint would have no effect on `json,css,scss,md` files anyway. Splitting the definitions I felt made it more clear as to what packages are being run on which files